<launch>
    <!-- t265 robot 1 -->
    <include file="$(find realsense2_camera)/launch/rs_t265.launch">
        <arg name="camera" value="cam1"/>
        <arg name="serial_no" value="929122110980"/>
    </include>
    <include file="$(find realsense2_camera)/launch/rs_t265.launch">
        <arg name="camera" value="cam2"/>
        <arg name="serial_no" value="2322110456"/>
    </include>

</launch>
